test/soak/serve_hostnames/serve_hostnames.go

```diff
@@ -44,6 +44,7 @@ var (
 	podsPerNode    = flag.Int("pods_per_node", 1, "Number of serve_hostname pods per node")
 	upTo           = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
 	maxPar         = flag.Int("max_par", 500, "Maximum number of queries in flight")
+	gke            = flag.String("gke", "", "Taget GKE cluster with context gke_{project}_{zone}_{cluster-name}")
```

Comment:
I thought about that. I problem with that is that if you provide a
--context argument for a GCE cluster then things won't work? Right?

On Wed, Apr 1, 2015 at 7:50 PM, Robert Bailey <notifications@github.com>
wrote:

> In test/soak/serve_hostnames/serve_hostnames.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6333#discussion_r27629575>
> :
>
> > @@ -44,6 +44,7 @@ var (
> >  	podsPerNode    = flag.Int("pods_per_node", 1, "Number of serve_hostname pods per node")
> >  	upTo           = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
> >  	maxPar         = flag.Int("max_par", 500, "Maximum number of queries in flight")
> > +	gke            = flag.String("gke", "", "Taget GKE cluster with context gke_{project}_{zone}_{cluster-name}")
>
> It'd be more generic to make a --context flag but as a GKE user I
> appreciate making this particular case nice and easy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6333/files#r27629575>
> .
>